src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_time_field/__tests__/step_time_field.test.js

```diff
@@ -68,11 +68,82 @@ describe('StepTimeField', () => {
         { display: 'name', fieldName: 'name' },
       ],
       selectedTimeField: '@timestamp',
+      timeFieldSet: true,
     });
 
     expect(component).toMatchSnapshot();
   });
 
+  it('should ensure disabled time field options work properly', () => {
+    const component = shallow(
+      <StepTimeField
+        indexPattern="ki*"
+        indexPatternsService={indexPatternsService}
+        goToPreviousStep={noop}
+        createIndexPattern={noop}
+      />
+    );
+
+    component.setState({
+      timeFields: [
+        { display: '@timestamp', fieldName: '@timestamp' },
+        { display: 'name', fieldName: 'name' },
+      ],
+    });
+
+    component.instance().onTimeFieldChanged({ target: { value: undefined } });
```

Comment:
Can we add a couple comments for each of these assertions to clarify their context and intention? E.g. "An undefined time field means the user wants to create an index pattern without a time field", and "An empty string time field means the user hasn't specified one yet".